Samples-Typescript/Dashboard/PdfViewer/pdfviewer.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>PDF Viewer</title>
+
+    <!-- jQuery -->
+    <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+
+    <script src="../../../lib/tableau.extensions.1.latest.js"></script>
+
+    <!-- Our extension's code -->
+    <link rel="stylesheet" href="pdfviewer.css" />
+    <script src="../../../dist/pdfViewer.js"></script>
```

Comment:
where is pdfViewer.js?

Reply:
It's created as part of the build process when compiling typescript files.  All of the compiled typescript files are put in this project's dist folder per settings in the project's [webpack.config.js](https://github.com/tableau/extensions-api/blob/main/webpack.config.js).  Since it's created as part of the build process, it's ignored and not checked in per [.gitignore](https://github.com/tableau/extensions-api/blob/main/.gitignore#L8-L9) settings.  You'll see other typescript examples in this project do a similar approach like the [Parameters example](https://github.com/tableau/extensions-api/blob/main/Samples-Typescript/Dashboard/Parameters/parameters.html#L21).